hsv_detecter:
  reduction: 2
  h_min: 175
  h_max: 5
  s_min: 100
  s_max: 230
  v_min: 100
  v_max: 230
rects_tracker:
  size_weight: 4
  max_distance: 400
  min_width: 10
  min_height: 10
  min_age: 5
  category: red_can
#s4_sensor_detect_geometry:
s4_sensor_detect_merger:
  distance_max: 0.10
  obstacle_height: 0.06